genes/hgvs/hgvs_matcher.py

```diff
@@ -513,6 +513,8 @@ def clean_hgvs(self, hgvs_string) -> Tuple[str, List[str]]:
         cleaned_hgvs = cleaned_hgvs.replace("::", ":")  # Fix double colon
         if cleaned_hgvs[0:2].upper() == "M_":
             cleaned_hgvs = "NM_" + cleaned_hgvs[2:]
+        if cleaned_hgvs[0:2].upper() == "C_":
+            cleaned_hgvs = "NC_" + cleaned_hgvs[2:]
```

Comment:
I should have added there's also NR_ transcripts, so best to fix all thse at once.
Maybe check if
```
cleaned_hgvs[0:2].upper() in ("M_", "C_", "R_")
```
and if so, prefix an N to the start. No need to do `"NC_" + cleaned_hgvs[2:]` when you can just do `"N" + cleaned_hgvs`

Reply:
For reference, here's an example of an NR
```
NR_001566.1(TERC):n.427_428insC
```